src/app/app-components/breadcrumbs-navbar/breadcrumbs-navbar.component.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../styles.scss';
+
+.Rectangle {
```

Comment:
Hi Jonatan!

First comment of the mentoring :D

This is just cosmetics, I would recommend you to always name css classes using lower case.

Reply:
Got it! I will address this in next PR, - Thanks for the advice! :smiley: